dashboard/app/controllers/home_controller.rb

```diff
@@ -34,11 +34,16 @@ def health_check
 
   GALLERY_PER_PAGE = 5
 
-  # Signed in: redirect to /home
+  # Signed in student with assigned course: redirect to current_lesson
+  # Signed in not student with assigned course: redirect to /home
```

Comment:
Maybe some commas to clarify the logic?  Like "signed in, not student with assigned course"? :)